### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by over-geek**
## Code Review

In this milestone, I:
- Copied my code from the project that I completed during the Admissions Trial
- Added linters configuration files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      </div>
+      <nav id="nav-bar"><ul>
+        <li class="nav-link"><a href="#services">Features</a></li>
+        <li class="nav-link"><a href="#">How It works</a></li>
+        <li class="nav-link"><a href="#">Pricing</a></li>
+      </ul></nav>
+    </header>
+    <main>
+      <section id="header-content">
+        <h2>Handcrafted, home-made masterpieces</h2>
```
Comment (over-geek) [inline index.html:24]: I think it would be better to use an h1 header tag for this section 😉

**Review by over-geek — COMMENTED** [review]
# You are almost there 😎 
## Changes required ♻️ 

Fix all linter errors in your code

Please check my comments below and make all required changes

Submit for a re-review when you're done 😉 

Thank you 😁

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          <h2><img src="./delivery.svg" alt="">Fast Shipping</h2>
+          <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.  </p>
+        </li>
+        <li>
+          <h2><img src="./pencil-edit.svg" alt="">Quality Assurance</h2>
+          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+        </li>
+          
+      </ul>
+    </div>
```
Comment (over-geek) [inline index.html:49]: Please wrap these lines of code in a section tag. Using semantic tags accurately describes the content that is on the page.

File: index.html (hunk 1 above)
Comment (over-geek) [inline index.html:24]: I think it would be better to use an h1 tag for the text in the main section 😉

Comment (souad988) [reply index.html:24]: Good job @over-geek , your comment is next to the adequate line of code and your message is clear.
[ OPTIONAL ] it's good to add links or references to a document explaining why they should do so this way the student knows that the comment is not a personal point of view .

**Review by souad988 — APPROVED** [review]
Hi @over-geek , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
GitHub review feature used ✔️
Inline comments used ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**